src/data/certif.rs

```diff
@@ -516,3 +516,104 @@ impl RealmRoleCertificate {
         self.0.role.map(RealmRole)
     }
 }
+
+#[pyclass]
+pub(crate) struct SequesterAuthorityCertificate(
+    pub libparsec::types::SequesterAuthorityCertificate,
+);
+
+crate::binding_utils::gen_proto!(SequesterAuthorityCertificate, __repr__);
+crate::binding_utils::gen_proto!(SequesterAuthorityCertificate, __richcmp__, eq);
+
+#[pymethods]
+impl SequesterAuthorityCertificate {
+    #[new]
+    fn new(timestamp: DateTime, verify_key_der: SequesterVerifyKeyDer) -> Self {
+        Self(libparsec::types::SequesterAuthorityCertificate {
```

Comment:
It's just a matter of style but I noticed that it may be easier to read if we import `libparsec::types` then we can just do `types::SequesterAuthorityCertificate`. What do you think?

Reply:
I don't think it's more easy, types can be also a module in this crate